tron/core/job.py

```diff
@@ -89,6 +90,9 @@ def __init__(
         self.queueing = queueing
         self.all_nodes = all_nodes
         self.enabled = enabled
+        # config_enabled is equivalent to the setting in file, whereas enabled
+        # is the current enable setting
+        self.config_enabled = enabled if config_enabled is None else config_enabled
```

Comment:
When a job is created, the `enabled` kwarg comes from the config, so we should always set config_enabled = enabled. That way, even if tronctl changes self.enabled, config_enabled will still reflect the original config value.

You also don't need to save it in state because it just comes from the config.

Reply:
Originally, I believed that we didn't need to save config_enabled in state; however, there is a relevant edge case: 
1. Create job from config, enabled=False.
2. Turn off trond, enabled=False saved to state.
3. Change the config, enabled=True.
4. Turn on trond again.
    - No config_enabled: trond would recreate the job from config (enabled=True), then overwrite it from saved state (enabled=False). Here, tronfig will not acknowledge a config change that happens offline.
    - With config_enabled: trond would recreate the job from config (enabled=True, config_enabled=True), then attempt to overwrite it from state (enabled=False, config_enabled=False). It would see the difference in config_enabled between config and state, and therefore keep enabled=True.

Reply:
This is an interesting edge case... I don't think you need to worry about it, because it is not possible to change the config while trond is off in any "normal" way. You can only do that by manually overwriting files in Tron's working directory.